plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestingSqlServer.java

```diff
@@ -23,8 +23,11 @@
 import static java.lang.String.format;
 
 public final class TestingSqlServer
-        extends MSSQLServerContainer<TestingSqlServer>
+        implements AutoCloseable
```

Comment:
This is `AutoCloseable` and `TestingMySqlServer` is `Closeable`. Why the difference?